### ISSUE TITLE
Handle accessibility issues related to <a> tags

### ISSUE THREAD

**Issue by bran-van-d** [HubSpot/cms-theme-boilerplate#497]
### Description
[We currently remove the underline for `<a>` tags by default](https://github.com/HubSpot/cms-theme-boilerplate/blob/main/src/fields.json#L404-L434) which is leading to an accessibility issue, ["Links must be distinguishable without relying on color"](https://dequeuniversity.com/rules/axe/4.8/link-in-text-block?application=AxeChrome). The link on the page isn't actually a different color from the default, but the lack of the underline is what creates the issue -- adding the underline back allows the link to be properly distinguished from surrounding text, passing this requirement.

We should let `<a>` tags have an underline by default, as users have the option to remove or otherwise edit the `<a>` to their liking.
